### PR TITLE
[otl] Update to version 4.0.451

### PR DESCRIPTION
**Pull request by c72578**
The current version of otl is now also available with a filename including the version, e.g. otlv4_40451.zip [1, 2]

- Use variable `OTL_VERSION` in portfile
- Rename header file to `otlv4.h`
  This is the current default name of the header file and typically included using:
  `#include "otlv4.h"` or `#include <otlv4.h>`

[1] https://sourceforge.net/p/otl/discussion/208859/thread/fdb9d3bacc/#45bf
[2] http://otl.sourceforge.net/otl3_down.htm

**Describe the pull request**

- What does your PR fix? Fixes issue #
Update otl to the current version 4.0.451
Simplify portfile by using a variable for the version
- Which triplets are supported/not supported? Have you updated the CI baseline?
All triplets are supported. CI-baseline: NA
- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

### CODE REVIEW

**Comment by JackBoosY** [comment]
```
2020-01-04T12:18:04.9047756Z The following files are already installed in C:/vsts/_work/3/s/installed/x86-windows and are in conflict with stb:x86-windows
2020-01-04T12:18:04.9049652Z 
2020-01-04T12:18:04.9051732Z Installed by nanovg:x86-windows
2020-01-04T12:18:04.9053626Z     include/stb_image.h
```
Could you add `stb:x86-windows=skip` to _VCPKG_PATH/scripts/ci.baseline.txt_ ?

Thanks.

File: ports/otl/portfile.cmake [hunk 1]
```diff
@@ -1,9 +1,10 @@
 include(vcpkg_common_functions)
```
Comment (JackBoosY) [inline ports/otl/portfile.cmake]: We no longer need to include this, please remove this line.

File: ports/otl/portfile.cmake [hunk 2]
```diff
@@ -12,5 +13,5 @@ vcpkg_extract_source_archive_ex(
     NO_REMOVE_ONE_LEVEL
 )
 
-file(INSTALL ${SOURCE_PATH}/otlv40448.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/otl)
-file(INSTALL ${SOURCE_PATH}/otlv40448.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/otl RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/otlv${OTL_VERSION}.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/otl RENAME otlv4.h)
+file(INSTALL ${SOURCE_PATH}/otlv${OTL_VERSION}.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/otl RENAME copyright)
```
Comment (JackBoosY) [inline ports/otl/portfile.cmake]: Could you change this line to:
```
file(INSTALL ${SOURCE_PATH}/otlv${OTL_VERSION}.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```

Thanks.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/otl/portfile.cmake (hunk 1 above)
Comment (c72578) [reply ports/otl/portfile.cmake]: OK, removed.

File: ports/otl/portfile.cmake (hunk 2 above)
Comment (c72578) [reply ports/otl/portfile.cmake]: OK, changed.

**Comment by JackBoosY** [comment]
Could you please resolve theses conflicts?

Thanks,

**Comment by c72578** [comment]
OK, I have resolved the conflicts now.
Background info:
In the meantime, `otl` has been partly updated in master.
See: 9c860988ab67312761246b9d55242ee995596c1b and PR: #9107 
The following changes of this PR are already in master now and have been removed from the commit message here:
- Use variable OTL_VERSION in portfile
- Rename header file to otlv4.h

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**